dataland-frontend/tests/component/components/forms/UploadFieldSpecificReports.cy.ts

```diff
@@ -0,0 +1,176 @@
+import CreateSfdrDataset from "@/components/forms/CreateSfdrDataset.vue";
```

Comment:
Do we need one test for SRDR and one for P2P?

Reply:
Yes. Because we had two different bugs in both and I think this is one of our rules. Bug => Test for bug such that it never occurs again.